wp-content/plugins/core/src/Theme/Config/Web_Fonts.php

```diff
@@ -0,0 +1,134 @@
+<?php
```

Comment:
I think @faction23 is in the process of removing our webfont loader JS-based implementation here. We'll just have to be aware of merge conflicts when we get there.

Reply:
Yes i am doing a fair amount of work on these theme files right now